modules/standard/IO.chpl

```diff
@@ -1291,10 +1291,12 @@ proc file.check() throws {
 /* Return a syserr through out error if a file is invalid */
 proc file.check(out error:syserr) {
   var err: syserr = ENOERR;
-  try! {
+  try {
     check();
   } catch e: SystemError {
     err = e.err;
+  } catch {
```

Comment:
Since we don't do type checking on the kinds of errors we can receive, a catch-all block is needed to prevent the compiler from squawking at us. The other approach is to use a `try!`, but that's what this change is moving away from.